php.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    class Deposit {
+
+        /**
+         * the deposit ownner.
+         *
+         * @var string witout info
+         */
+        private $_owner;
+
+        /*
```

Comment:
Typo inside comment in word `ere`. Possible corrections:  `pre` (27), `re` (11), `were` (3)
Typo inside comment in word `coment`. Possible corrections:  `cement` (63), `comment` (51), `moment` (44)